README.md

```diff
@@ -4,27 +4,36 @@ Copyright (C) 2014 Google Inc.
 
 A simple Python VM Runtime 'hello world' example, which accesses the 'Users' service, and shows how to get information about the current instance.
 
-## Project Setup, Installation, and Configuration
+You can run this application only on the [App Engine VM Runtime][1]. Currently, the VM Runtime is in Beta, and as such may be subject to backward-incompatible changes.
 
-You can run this application only on the [App Engine VM
-Runtime][1]. Currently, the VM Runtime is part of the Early Access Program, and
-only the participants are able to run this application.
+## Project Setup
 
-## Deploying
+Create a billing enabled project and install the Google Cloud SDK as described [here](https://cloud.google.com/appengine/docs/python/managed-vms/#install-sdk) (this includes [installing Docker](https://cloud.google.com/appengine/docs/python/managed-vms/#install-docker))
+
+### Installing boot2docker on Linux (Workaround)
```

Comment:
Let's leave off the 'workaround' part - AFAIK this is the most straightforward way to do it. (And, I believe for MVMs, boot2docker is required, so they do want to do this).